content/docs/rfcs/template/README.md

```diff
@@ -0,0 +1,83 @@
+---
+slug: XX
+title: XX/(WAKU2|LOGOS|CODEX|*)-TEMPLATE
+name: (Waku v2 | Logos | Codex) RFC Template
+status: (raw|draft|stable)
+category: (Standards Track|Informational|Best Current Practice)
+tags: an optional list of tags, not standard
+editor: Daniel Kaiser <danielkaiser@status.im>
+contributors:
+---
+
+# (Info, remove this section)
```

Comment:
I wonder, should this be a template or in itself a spec? :D

Reply:
I could remove this and put things into COSS exclusively.
However, Imo, like this the template is self-contained and I would add the info to COSS as well anyways.
(see my comment above). Could also change this in a future PR. Wdyt?